VAMobile/src/api/secureMessaging/downloadFileAttachment.tsx

```diff
@@ -0,0 +1,37 @@
+import FileViewer from 'react-native-file-viewer'
+
+import { useQuery } from '@tanstack/react-query'
+
+import { SecureMessagingAttachment } from 'api/types'
+import { downloadFile, unlinkFile } from 'utils/filesystem'
+
+import { secureMessagingKeys } from './queryKeys'
+
+/**
+ * Fetch user message signature
+ */
+const downloadFileAttachment = async (file: SecureMessagingAttachment): Promise<boolean | undefined> => {
+  const filePath = await downloadFile('GET', file.link, file.filename)
+  if (filePath) {
+    await FileViewer.open(filePath, {
+      onDismiss: async (): Promise<void> => {
+        await unlinkFile(filePath)
+      },
+    })
+    return true
```

Comment:
actually true is needed

I tried it the other way and errors were thrown because it would return a void instead, frustrating but minor detail with using FileViewer